src/CMakeLists.txt

```diff
@@ -80,6 +80,7 @@ set(mudlet_SRCS
     KeyUnit.cpp
     LuaInterface.cpp
     main.cpp
+    mapInfoContributorManager.cpp
```

Comment:
Contributor or provider? The latter is also used often.

I'd go with contributor and keep it simple with `mapInfoContributors.cpp`.